Stripe/STPCardParams.m

```diff
@@ -53,6 +53,30 @@ - (void)setAddress:(STPAddress *)address {
     self.addressCountry = address.country;
 }
 
+#pragma mark - Description
+
+- (NSString *)description {
+    NSArray *props = @[
+                       // Object
+                       [NSString stringWithFormat:@"%@: %p", NSStringFromClass([self class]), self],
+
+                       // Basic card details
+                       [NSString stringWithFormat:@"last4 = %@", self.last4],
+                       [NSString stringWithFormat:@"expMonth = %lu", (unsigned long)self.expMonth],
+                       [NSString stringWithFormat:@"expYear = %lu", (unsigned long)self.expYear],
+                       [NSString stringWithFormat:@"cvc = %@", (self.cvc) ? @"<redacted>" : nil],
```

Comment:
Seems like it might be worth pulling this into a helper, maybe a private `NSObject` category with `stp_redactedDescription`?